CMakeLists.txt

```diff
@@ -633,6 +778,17 @@ if (LIBRAW_INSTALL)
     install(FILES ${CMAKE_CURRENT_SOURCE_DIR}/cmake/modules/FindLibRaw.cmake
             DESTINATION ${CMAKE_INSTALL_DATADIR}/cmake/libraw)
 
+    # Install DNG SDK's host header as downstream clients may need it
```

Comment:
Is this standard practice for the DNG SDK? Normally, a library should only ship its own headers.

Reply:
> Is this standard practice for the DNG SDK? Normally, a library should only ship its own headers.

No, I think we would have to move this to an install step for the DNG SDK. The reason being is downstream libraries need the "dng_host.h" in order to configure libraw to use the Adobe SDK as you need to pass it an instance of the dng  host class.